### PR TITLE
de (German) translation

### PR DESCRIPTION
**Pull request by LeSnake04**
Commit made via [Stringlate](https://lonamiwebs.github.io/stringlate/)

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="app_info">App-Info</string>
+    <string name="random_colors">Zufällige Farben</string>
+    <string name="reset_to_default">Zurücksetzen</string>
+    <string name="freeze_size">Größe einfrieren.</string>
+    <string name="hide">Verstecken</string>
+    <string name="unfreeze_size">Größe nicht mehr Einfrieren</string>
+    <string name="hidden_apps">Versteckte Apps</string>
+    <string name="freezed_apps">Eingefrorene Apps</string>
+
+    <string name="restore">Wiederhergestellen</string>
```
Comment (SubhamTyagi) [inline app/src/main/res/values-de/strings.xml]: Is this correct ?

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="fixed_colors">Feste Farben</string>
+    <string name="freeze_apps_size">Appgrößen einfrieren</string>
+    <string name="unfreeze_app_size">Appgröße nicht mehr einfrieren</string>
+    <string name="default_theme">Standartdesign</string>
```
Comment (SubhamTyagi) [inline app/src/main/res/values-de/strings.xml]: is this right?

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="background_wallpaper">Hintergrundbild</string>
+    <string name="white_on_black">Weiß auf Schwarz</string>
+    <string name="black_on_white">Schwarz auf Weiß</string>
+    <string name="white_on_grey">Weiß auf Grau</string>
+    <string name="black_on_grey">Schwarz auf Grau</string>
+    <string name="hacker_green">Hackergrün</string>
```
Comment (SubhamTyagi) [inline app/src/main/res/values-de/strings.xml]: is this correct?

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    <string name="hacker_red">Hacherrot</string>
```
Comment (SubhamTyagi) [inline app/src/main/res/values-de/strings.xml]: is this right?

i did Google translate and found the same result.

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (LeSnake04) [reply app/src/main/res/values-de/strings.xml]: Wiederhergestellen = Wiederherstellen

File: app/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (LeSnake04) [reply app/src/main/res/values-de/strings.xml]: *Hackerrot

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (SubhamTyagi) [reply app/src/main/res/values-de/strings.xml]: > Standartdesign
is this correct?

**Comment by SubhamTyagi** [comment]
Could you change these strings?

**Comment by SubhamTyagi** [comment]
@LeSnake04  Thanks for your contribution.

**Pull request merged**